app/models/contributor.rb

```diff
@@ -69,11 +69,9 @@ def self.with_lowercased_email(email)
   end
 
   def reply(message_decorator)
-    message = message_decorator.message
-    request = message.request || active_request
-    return unless request
-
+    request = active_request or return nil
```

Comment:
```suggestion
    request = active_request 
    return nil unless request
```
Looks a bit more idiomatic.

`and` and `or` in ruby have low precedence. It won't matter here but with the suggested change it's more obvious that the assignment gets executed first and we return if `request` is nil, not if the return value of the assignment is nil.
![image](https://github.com/user-attachments/assets/451ae699-a77a-407e-9f47-4f99ca3f4864)

Reply:
I like this change, I also implemented it this way when I was adding the reply to logic to this method here https://github.com/tactilenews/100eyes/pull/2089/commits/5da7eb560a391990a25b51b1bd6b85b010862dec#diff-5147cd4a4f7646dbcf1c6e9ba4df962eb805e7fab7b9dbe6171e28ef6ac35c11R72-R74, but since I've reverted all changes to this file, I won't update it. My idea is to refactor so that the contributor class does not have the responsibility of assigning the correct request to a reply or saving the message and then this code will be removed.